src/components/Votes.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+import { patchVotes } from "./utils";
+
+const Votes = ({ votes, article_id }) => {
+  const [currentVotes, setCurrentVotes] = useState(votes);
+
+  const userVote = (inc) => {
+    setCurrentVotes(currentVotes + inc)
+
+    patchVotes(article_id, inc).then(({ data: { article: { votes } } }) => {
+      setCurrentVotes(votes);
+      console.log(votes);
+    });
+  };
```

Comment:
This isn't optimistically rendered please take a look at the notes on how you can change this so it's not reliant on the api call.